tests/e2e/support/commands.js

```diff
@@ -97,3 +97,43 @@ Cypress.Commands.add('addLineItem', (url, quantity) => {
   cy.get('[data-test=add-to-cart-form-button]').click();
   cy.get('[data-test=mini-cart-content]').should('be.visible');
 });
+
+Cypress.Commands.add('createMyOrder', (draft) => {
+  const createNewOrder = client => client.query({
+    query: gql`
+      query queryCustomerByEmail($predicate: String) {
```

Comment:
How do you know this customer exists? You should make sure the customer exists before querying it (by removing it if it exists, and creating it. i.e. calling cy.createCustomer).

Reply:
For now it's fine, as it's always logged in. But when working with orders without customer we may need a more generic thing. Let's solve that problem in the future when it arrives (if it does) :)